app/src/userpages/components/StreamPage/View.jsx

```diff
@@ -26,7 +26,7 @@ import StatusLabel from '$shared/components/StatusLabel'
 import Nav from '$shared/components/Layout/Nav'
 import Storage from './shared/Storage'
 
-const Text = styled(UnstyledText)`
+export const Text = styled(UnstyledText)`
```

Comment:
Why do we export those?

PS. Asking cause I may have missed something (GH's Viewed feature is dumb, I can't use search)…

Reply:
I reuse those components in the new stream page.